### ISSUE TITLE
jsk sync command broken on dpy master

### ISSUE THREAD

**Issue by Ken-Miles** [scarletcafe/jishaku#233]
## Summary

The sync command is broken when using the master version of discord.py (and soon discord.py 2.4).

## Reproduction steps

Use the master branch of discord.py: 
`discord.py @ git+https://github.com/Rapptz/discord.py@08b138cc71fea5594c068082ee565a05cde2c7fa`

Attempt to sync 
## Expected results

The syncing of my command tree

## Actual results

The sync command errors, giving the following traceback:

```
Ignoring exception in command jishaku sync:
Traceback (most recent call last):
  File "/home/me/bot/.venv/lib/python3.11/site-packages/discord/ext/commands/core.py", line 235, in wrapped
    ret = await coro(*args, **kwargs)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/j/bot/.venv/lib/python3.11/site-packages/jishaku/features/management.py", line 242, in jsk_sync
    payload = [command.to_dict() for command in slash_commands]
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/j/bot/.venv/lib/python3.11/site-packages/jishaku/features/management.py", line 242, in <listcomp>
    payload = [command.to_dict() for command in slash_commands]
               ^^^^^^^^^^^^^^^^^
TypeError: Command.to_dict() missing 1 required positional argument: 'tree'
```
## Checklist

- [X] I have updated discord.py and jishaku to the latest available versions and have confirmed that this issue is still present
- [X] I have searched the open issues for duplicates
- [X] I have shown the entire traceback, if possible
- [X] I have removed my token from display, if visible

## System information

JSK 2.5.2
discord.py @ git+https://github.com/Rapptz/discord.py@08b138cc71fea5594c068082ee565a05cde2c7fa

**Comment by Ken-Miles** [comment]
Right after posting this, I found a PR that supposedly fixes this: https://github.com/Gorialis/jishaku/pull/230

**Comment by munishkhatri720** [comment]
Same issue here
```py
Traceback (most recent call last):
  File "C:\Users\Administrator\Documents\Hade Manager\venv\Lib\site-packages\discord\ext\commands\core.py", line 235, in wrapped
    ret = await coro(*args, **kwargs)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Administrator\Documents\Hade Manager\venv\Lib\site-packages\jishaku\features\management.py", line 242, in jsk_sync
    payload = [command.to_dict() for command in slash_commands]
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Administrator\Documents\Hade Manager\venv\Lib\site-packages\jishaku\features\management.py", line 242, in <listcomp>
    payload = [command.to_dict() for command in slash_commands]
               ^^^^^^^^^^^^^^^^^
TypeError: Group.to_dict() missing 1 required positional argument: 'tree'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\Administrator\Documents\Hade Manager\venv\Lib\site-packages\jishaku\features\invocation.py", line 168, in jsk_debug
    await alt_ctx.command.invoke(alt_ctx)
  File "C:\Users\Administrator\Documents\Hade Manager\venv\Lib\site-packages\discord\ext\commands\core.py", line 1650, in invoke
    await ctx.invoked_subcommand.invoke(ctx)
  File "C:\Users\Administrator\Documents\Hade Manager\venv\Lib\site-packages\discord\ext\commands\core.py", line 1029, in invoke
    await injected(*ctx.args, **ctx.kwargs)  # type: ignore
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Administrator\Documents\Hade Manager\venv\Lib\site-packages\discord\ext\commands\core.py", line 244, in wrapped
    raise CommandInvokeError(exc) from exc
discord.ext.commands.errors.CommandInvokeError: Command raised an exception: TypeError: Group.to_dict() missing 1 required positional argument: 'tree'
```

**Comment by bitterteriyaki** [comment]
Any updates on this?

**Comment by Ken-Miles** [comment]
Still experiencing this issue even with 2.4. Had to make my own sync command until the PR is merged.

**Comment by Gorialis** [comment]
#235 should have fixed this. Try [installing a development version](https://github.com/Gorialis/jishaku?tab=readme-ov-file#installing-development-versions) and see if it fixes it.

**Comment by bitterteriyaki** [comment]
> #235 should have fixed this. Try [installing a development version](https://github.com/Gorialis/jishaku?tab=readme-ov-file#installing-development-versions) and see if it fixes it.

Oops, I didn't see that this was fixed on master branch. Everything is working now. When will be there a new release?

**Comment by scarletcafe** [comment]
Closed by #212
